ios/podverse/Info.plist

```diff
@@ -171,5 +173,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>LSSupportsOpeningDocumentsInPlace</key>
+	<false/>
```

Comment:
@kreonjr can you explain what these 2 were added for? It seems like LSHandlerRank may not have an impact since it's "Default"? And I'm not sure why we would need LSSupportsOpeningDocumentsInPlace...